chainerx_cc/chainerx/error.h

```diff
@@ -99,4 +99,10 @@ class GradientCheckError : public ChainerxError {
     using ChainerxError::ChainerxError;
 };
 
+// Error on using invalid contexts.
+class ValueError : public ChainerxError {
```

Comment:
Sorry but where is this exception thrown?

Reply:
Oops. Used to throw it previously. Now we don't. Will remove.